.github/workflows/contract_requiring_verification_published.yml

```diff
@@ -20,12 +26,17 @@ jobs:
   verify-contract-requiring-verification:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: checkout default branch if user manually provides pact URL
```

Comment:
This was really here, so that we could run allow people to run through the entire workflow without having to check the codebase out on their local machine (they can just trigger from GitHub Actions - with the caveat that this would traditionally be done on a developers machine) 

Ahh it may actually be dangerous to do this in retrospect, as it will be publishing the verification results.

I thought it might be useful, as the pre-req setups for the ci/cd guide suggest you clone the codebases to your machine, so if you are a Windows user, and you aren't using WSL, they are going to have to go down a rabbit hole installing Make, which always gets people, so this allows us to follow the format of the quick start guides

https://docs.pactflow.io/docs/workshops/quick_starts/bdc

also just noticed the url says bdc not bdct 😭 😂